lib/copyable/saver.rb

```diff
@@ -1,19 +1,27 @@
 module Copyable
   class Saver
 
+    def self.direct_sql_insert!(new_model)
+      new_model.send(:_create_record) # bypass all callbacks and validations
```

Comment:
Ah, yeah, that's hacky. Not quite as aggressive as duck-punching, but with a private API call like this we run the risk of the API changing under us with no notice. I'm not saying we should change it, but we may need to comment/README and be pretty careful about our AR version pins.

Reply:
yes, i've read the rails code pretty carefully. fortunately, in future versions of rails there will actually be a public api for insert/upsert which does what you think it does: just puts stuff in the db with no callbacks/etc. Compare [4.2](https://github.com/rails/rails/blob/v4.2.11/activerecord/lib/active_record/persistence.rb#L484) with [master](https://github.com/rails/rails/blob/master/activerecord/lib/active_record/persistence.rb). Notice that `_update_record` and `_create_record` are both still there, too. I think this is a bit more stable than before.